websites/mswjs.io/src/content/docs/recipes/using-local-https.mdx

```diff
@@ -33,6 +33,10 @@ You can add your self-signed certificate to the system's trusted certificates. P
 
 > If encountering certificate fetching/validation issues, set the `ignore-certificate-errors` flag to "Enabled".
 
+### Chrome on Windows
```

Comment:
Wouldn't already present instructions for Chrome also work on Windows? I suspect it ships with the same flags so you can set `unsafely-treat-insecure-origin-as-secure` as well.